engines/e_dasync_err.h

```diff
@@ -1,15 +1,16 @@
 /*
  * Generated by util/mkerr.pl DO NOT EDIT
- * Copyright 1995-2019 The OpenSSL Project Authors. All Rights Reserved.
+ * Copyright 1995-2020 The OpenSSL Project Authors. All Rights Reserved.
  *
  * Licensed under the Apache License 2.0 (the "License").  You may not use
  * this file except in compliance with the License.  You can obtain a copy
  * in the file LICENSE in the source distribution or at
  * https://www.openssl.org/source/license.html
  */
 
-#ifndef OSSL_ENGINES_E_DASYNC_ERR_H
-# define OSSL_ENGINES_E_DASYNC_ERR_H
+#ifndef OSSL_E_DASYNC_ERR_H
+# define OSSL_E_DASYNC_ERR_H
```

Comment:
Could the renaming of this define and other similar ones be considered an API break?

Reply:
Why?  These headers were never installed anywhere...  and this is just a guard for this file.  At some point, they were called things like `HEADER_CAPI_ERR_H`, that got changed in 1.1.1 (!)